gitpod-io/website
# ✨ Revamp `/dedicated` Page

## Description



- This PR Revamps `/dedicated` page
- This PR also changes this particular component to align the size with Figma Design
    |Before|After|
    |:---:|:---:
    |![image](https://github.com/gitpod-io/website/assets/55068936/533d88bd-aa2d-4e95-a457-be233e7e57c7)|![image](https://github.com/gitpod-io/website/assets/55068936/e15eff0d-e8d9-4666-b5b3-bced1d08b1d0)|

## Related Issue(s)



Fixes MKWEB-34

## How to test


- Open Preview
- Go to `/dedicated` & see 👀 

## Release QA Checklist ✔️



<details>
  <summary>QA Checklist </summary>

## [Priority: High] Functional Testing



## [Priority: High] Content Testing


## [Priority: Medium] Performance Testing


## [Priority: Medium] Accessibility Testing


By including these key items in your website release QA checklist, you can ensure that your website is fully functional, user-friendly, and meets your business requirements.

## Security Testing

> **Note**:💡 If we are changing something related to cookies or data storing or analytics side.


  </details>
